### PR TITLE
update cleanup to run every 4hours and only remove week old containers

### PR DESCRIPTION
**Pull request by cfortier2**

### CODE REVIEW

**Comment by bryanlatten** [comment]
week old might be a....lot, no? especially since almost any host can run any container

**Comment by cfortier2** [comment]
my thinking was that if theres an issue at 8pm on a friday, we'd want to be able to look at it on monday morning. so that gives us like 3 days. but trying to figure out 3 days in bash isn't fun, but 1 week is easy. 

but i can work it more so that we keep a couple of days.

**Comment by bryanlatten** [comment]
ok, i'm only worried about the immediate disk IO as it removes a full week of containers. can we set it to a max that it would try to remove?

File: v1/fleet_units/docker-cleanup.service [hunk 1]
```diff
@@ -9,15 +9,16 @@ Restart=on-failure
 RestartSec=20
 TimeoutStartSec=0
 ExecStart=/usr/bin/sh -c ' \
-  if [[ $(/usr/bin/docker ps -a | grep Exited | wc -l) -ne 0 ]]; \
+  if [[ $(/usr/bin/docker ps -a | grep Exited | awk \'{print $1 "\\t" $7 "\\t" $10}\' | grep week | wc -l) -ne 0 ]]; \
   then \
     echo "starting docker cleanup"; \
-    /usr/bin/docker ps -a | grep Exited | awk \'{print $1}\' | xargs /usr/bin/docker rm ; \
+    /usr/bin/docker ps -a | grep Exited | awk \'{print $1 "\\t" $7 "\\t" $10}\' | grep week | awk \'{print $1}\' | head -n25 | xargs /usr/bin/docker rm -v ; \
```
Comment (bryanlatten) [inline v1/fleet_units/docker-cleanup.service]: can change the output to `starting docker cleanup, 25 max`? (using a new variable, to be defined above)

Comment (cfortier2) [inline v1/fleet_units/docker-cleanup.service]: yes. added.

**Comment by bryanlatten** [comment]
:+1:

**Comment by bryanlatten** [comment]
lookin good!

**Pull request merged**